README.md

```diff
@@ -0,0 +1,15 @@
+# Using plugins
+Plugins can be used to increase functionality in neo-cli. One common example is to add the ApplicationLogs plugin in order to enable your node to same log files.
```

Comment:
> Plugins can be used to increase functionality in neo-cli.

In fact, you can increase the functionality of both neo-cli and neo-gui (and any projects that use neo.dll).

Reply:
Do you think this should be added here instead: https://github.com/neo-project/neo ?
I think we should reference it in both neo-cli and neo-gui because this are the projects that we use, since we use neo as a nuget dependency. May I change the text and add it to neo-gui too?